### PR TITLE
docs: DLT-1897 move storybook to doc site - Emoji to Emoji Text Wrapper

### PR DESCRIPTION
**Pull request by ninamarina**
# Move storybook to doc site - Emoji to Emoji Text Wrapper

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket
[DLT-1897]

## :book: Description
Move SB to doc site:

- Emoji (updated)
- Emoji picker (fixed and updated)
- Emoji text wrapper (fixed custom emojis and updated)

Direct link to the pages that changed:
https://dialtone.dialpad.com/deploy-previews/pr-412/components/emoji.html
https://dialtone.dialpad.com/deploy-previews/pr-412/components/emoji-picker.html
https://dialtone.dialpad.com/deploy-previews/pr-412/components/emoji-text-wrapper.html

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

[DLT-1897]: https://dialpad.atlassian.net/browse/DLT-1897?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/dialtone-documentation/docs/components/emoji-text-wrapper.md [hunk 1]
```diff
@@ -47,6 +57,77 @@ vueCode='
...
 <component-vue-api component-name="emojitextwrapper" />
+
+## Usage
+
+### Import
+
+```javascript
+// Note you must import from @dialpad/dialtone-vue. The emoji module is not included in the main
+// @dialpad/dialtone-vue package due to it's large size. You may wish to consider dynamically importing
+// the emoji module in your application.
```
Comment (juliodialpad) [inline apps/dialtone-documentation/docs/components/emoji-text-wrapper.md]: This is no longer a true statement, emoji-text-wrapper is now part of dialtone-vue

**Review by juliodialpad — CHANGES_REQUESTED** [review]
Looking good! small change and some comments left:

Can we center this emojis please? https://dialtone.dialpad.com/deploy-previews/pr-412/components/emoji.html#sizes


image

**Comment by francisrupert** [comment]
> Can we center this emojis please?

Hmm, this actually looks like a bug to me. They technically **are** centered but the rendered image within `.d-emoji` shifts out of its bounding container needs to be block-level to fill the space correctly. 

And of course, ensure this doesn't impact the Emoji Wrapper PR @juliodialpad currently has going. 

https://github.com/user-attachments/assets/43ffbe45-5e2c-405b-8416-cafdb2e8d228

**Comment by braddialpad** [comment]
@ninamarina let's merge in staging, now that Julio's emoji alignment change is in there.

**Comment by braddialpad** [comment]
Similar alignment issues here as well:

Screenshot 2024-07-24 at 1 22 20 PM

File: apps/dialtone-documentation/docs/components/emoji.md [hunk 2]
```diff
@@ -40,6 +48,188 @@ vueCode='
...
 <component-vue-api component-name="emoji" />
+
+## Assets
+
+Dialtone Vue uses [JoyPixels 8.0](https://joypixels.com/) to render emojis. When using this component, Dialtone Vue will use the free joypixels assets hosted on jsdelivr CDN by default. You may wish to use self hosted joypixels assets such as the SVGs only available to premium license holders. The examples shown here use the free 32px PNGs for emojis 16px and under, and the premium SVGs for emoji's larger than 16px. You may set a custom URL for small emojis and large emojis with the following functions during initialization of your app:
+
+```javascript
+// Note you must import from @dialpad/dialtone-vue. The emoji module is not included in the main
+// @dialpad/dialtone-vue package due to it's large size. You may wish to consider dynamically importing
+// the emoji module in your application.
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/emoji.md]: Same thing Julio mentioned but in emoji.md file. Can remove this comment

**Comment by ninamarina** [comment]
To fix the alignment, I applied the same that Julio did for the emoji text wrapper in the scoped styles of this page. I am assuming putting all sizes of emojis one after the other is not a real use case.

**Comment by francisrupert** [comment]
> To fix the alignment, I applied the same that Julio did for the emoji text wrapper in the scoped styles

I think this actually needs to be fixed at the item level itself. The rendered emoji `` isn't properly sitting in its bounding container. 

image

**Comment by ninamarina** [comment]
> > To fix the alignment, I applied the same that Julio did for the emoji text wrapper in the scoped styles
> 
> I think this actually needs to be fixed at the item level itself. The rendered emoji `` isn't properly sitting in its bounding container.
> 
> image

I created this ticket to fix that: https://dialpad.atlassian.net/browse/DLT-1913
This one is just for moving the docs, so we should address that as a separate bug.

**Review by juliodialpad — APPROVED** [review]
Looking good, thanks Nina!

**Review by braddialpad — APPROVED** [review]
Good by me as well.

**Pull request merged**